html/time-settings.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Time Settings</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+</head>
+
+<body>
+    {{template "navbar"}}
+
+    <div class="container">
+        {{if ne .ErrorMessage ""}}
+        <div class="alert alert-danger">
+            {{.ErrorMessage}}
+        </div>
+        {{end}}
+
+        {{if ne .Message ""}}
+        <div class="alert alert-success">
+            {{.Message}}
+        </div>
+        {{end}}
+
+        <div class="container pt-5 pl-0">
+            <h2>Time Settings<br></h2>
+        </div>
+        <hr>
+
+        <form action="/time-settings" method="POST">
+          <div class="form-row">
+            <div class="form-group col-md-6">
+              <label for="rtcdate">RTC Date</label>
```

Comment:
Can you change all references to "RTC" to "Hardware Clock"? Not everyone will understand what RTC means.